### ISSUE TITLE
Inference values in C# and pytorch are different.

### ISSUE THREAD

**Issue by Sun-Pak** [microsoft/onnxruntime#17205]
### Describe the issue

I extracted BertForTokenClassification model from pytorch as onnx type.
To import it into Unity, useing ML.Onnxruntime and Onnxruntime.tensor.
Instead of using the Nuget Pack Package, I directly downloaded the dll file and put it into unity.(Because there is a version problem)
Looking at this(https://youtu.be/7gDrt-SQZc8), I made a code.
I used FsTokenizers.

### To reproduce

And this is my C#code

using Microsoft.ML.OnnxRuntime;
using Microsoft.ML.OnnxRuntime.Tensors;
using System;
using System.Collections.Generic;
using System.Linq;
using UnityEngine;
using UnityEngine.Windows;
using BERTTokenizer;

            string my_vocab_path = Application.dataPath + vocabPath;
            var vocab = Vocabulary.loadFromFile(my_vocab_path);

            string inferText = "attack red color golem";
            var ftrs = Featurizer.toFeatures(vocab, false, 64, inferText, "");
            List<int> ints = ftrs.InputIds.ToList();
            var longs = ints.ConvertAll(i => (long)i).ToArray();
            var input_ids = ConvertToTensor(longs, ftrs.InputIds.Length);

            var int_mask = ftrs.InputMask.ToArray();
            var long_mask = int_mask.Select(item => (long)item).ToArray();
            var attention_mask = ConvertToTensor(long_mask, long_mask.Length);

            var input = new List<NamedOnnxValue> { NamedOnnxValue.CreateFromTensor("input_ids", input_ids),
                                                    NamedOnnxValue.CreateFromTensor("attention_mask", attention_mask)
                                                     };

            var session = new InferenceSession(my_model_path);
            var output = session.Run(input);
            var resultsArray = output.ToArray();
            
            var outputTensor = output.First().AsTensor<float>(); 

            var batch_size = input_ids.Dimensions[1];

            var logits = outputTensor.Reshape(new[] { batch_size, 12 });

            var predict = new int[batch_size];

            for (int i = 0; i < batch_size; i++)
            {
                float maxLogit = logits[i, 0];
                int maxIndex = 0;
                for (int j = 1; j < 12; j++)
                {
                    if (logits[i, j] > maxLogit)
                    {
                        maxLogit = logits[i, j];
                        maxIndex = j;
                    }
                }

                predict[i] = maxIndex;
            }

When I ran my onnx model, the value of logits[0,0] was different from logits[0,0] in python.
The value of logits[0,0] in C# is   5.800
in pytorch is  6.0365

this is my python code.

   
    ids = seq['input_ids'].to(device)
    feed = dict(
        input_ids=np.array(seq["input_ids"]).astype("int64"),
        attention_mask=np.array(seq["attention_mask"]).astype("int64"),
    )
    ort_outs = model.run(None, feed)
    output_tensor = torch.tensor(ort_outs[0][0])
    logits = output_tensor.view(-1,12)
    
   predict = torch.argmax(logits, axis=1)
    token_predictions = [my_ids_to_labels[i] for i in predict.cpu().numpy()]
    token = tokenizer.convert_ids_to_tokens(feed["input_ids"].squeeze())

    wp_preds = list(zip(token, token_predictions))

### Platform

Windows

### OS Version

10

### ONNX Runtime Installation

Released Package

### ONNX Runtime Version or Commit ID

1.12.0

### ONNX Runtime API

C#

### Architecture

X64

### Execution Provider

CUDA

**Comment by Sun-Pak** [comment]
I solved It was a tokenizer. use BERTTokenizer from tutorial
